src/js/components/modals/AppModalComponent.jsx

```diff
@@ -115,6 +141,10 @@ var AppModalComponent = React.createClass({
     }
   },
 
+  hasInfoObject: function () {
```

Comment:
Shouldn't this be a "private" or util method as it validates that the store has some data and not that the component has the info object.